draft-group-privacypass-k-check.md

```diff
@@ -262,7 +262,29 @@ factors, including, but not limited to, the number of clients interacting with a
 mirror, whether or not the mirror is trustworthy, and application requirements for dealing
 with consistency check failures.
 
-<!-- TODO: weave in considerations here -->
+## Handling Consistency Failures
+
+If a consistency check fails, either because the request to the mirror failed or
+because the mirrored resource did not match, the client MUST NOT use the original resource.
+The client behavior for what to do instead can vary depending on the protocol, availability
+of alternative services, and client policy.
```

Comment:
```suggestion
If a consistency check fails because the mirrored resource did not match, the client
MUST NOT use the original resource. For cases where the check failed because the
client was unable to communicate with the mirror, client policy dictates whether or
not to assume the resource is consistent. Client behavior for what to do in the case
of inconsistency can vary depending on the protocol, availability of alternative services,
and client policy.
```